src/main/scala/io/citrine/lolo/trees/splits/Splitter.scala

```diff
@@ -17,4 +21,50 @@ object Splitter {
       Math.abs((x1 - x2) / (x1 + x2)) > 1.0e-9
     }
   }
+
```

Comment:
`RegressionSplitter`, `ClassificationSplitter`, and `MultitaskSplitter` all used essentially the same code for `getBestRealSplit`, so I centralized it here.